src/libraries/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -226,6 +226,10 @@
     <Compile Include="FileInfo\AppendText.cs" />
     <Compile Include="FileInfo\CopyTo.cs" />
     <!-- Helpers -->
+    <Compile Include="$(CommonPath)Interop\Windows\Interop.BOOLEAN.cs" Link="Interop\Windows\Interop.BOOLEAN.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Interop.Libraries.cs" Link="Interop\Windows\Interop.Libraries.cs" />
```

Comment:
What's the correct way to solve this? I can't include this file because it conflicts with the Unix one on Unix platforms. If I don't include this file, then I can't include `DeviceIoControl`, because it uses the `Kernel32` constant for its dll name. I could go back to having a local definition for `DeviceIoControl`.

Also, I don't need `BOOLEAN` and `FILE_SET_SPARSE_BUFFER` here I think, so will remove them.

Reply:
Include these files for the Windows build of the test only. Look for `Condition="'$(TargetPlatformIdentifier)' == 'windows'"` in this .csproj file.

Reply:
And then `#if` the test I assume

Reply:
You can put the Windows only tests in a file like foo.Windows.cs